### DISCUSSION TITLE
Convite para fazer parte da Angola Open-source Community Mentorship

### DISCUSSION THREAD

**Discussion by antonio-pedro99** [Respect/Validation#1433] [General]
Olá, equipe [Respect](https://github.com/Respect)

Espero que estejam bem. Sou Antonio Pedro da [Angola Open-source Community](https://github.com/angolaosc), uma comunidade de desenvolvedores dedicados a promover o desenvolvimento open-source em Angola.

Estamos atualmente organizando um Programa de Mentoria em Desenvolvimento Open-source em Angola, visando capacitar os nossos desenvolvedores locais e estimular a colaboração com projetos de código aberto de renome internacional.

Gostaríamos de convidar o [Respect](https://github.com/Respect) para participar como uma organização mentora no nosso programa. Acreditamos que a participação do [Respect](https://github.com/Respect) como mentor pode ser extremamente valiosa para nossos desenvolvedores, permitindo que eles contribuam ativamente para um projeto relevante e aprendam com as melhores práticas de uma organização experiente.

O nosso [programa](https://github.com/angolaosc/mentorship) terá duração de um mês e contará com diferentes etapas, incluindo pré-mentoria(onde nós a organização, iremos conduzir um workshop introdutório sobre open-source), mentoria, onde os desenvolvedores terão a oportunidade de trabalhar em qualquer issue aberta neste projeto, receber feedback de especialistas e desenvolver as suas habilidades em desenvolvimento de software.

Seria uma honra contar com o [Respect](https://github.com/Respect) como um mentor no nosso programa. Acreditamos que essa colaboração pode trazer benefícios mútuos, fortalecendo a comunidade open-source em Angola e promovendo a inovação tecnológica em ambos os lados.

Se estiverem interessados em participar ou tiverem-se alguma dúvida, por favor, deixem-nos saber. Estamos ansiosos para uma possível parceria com o [Respect](https://github.com/Respect).

Agradecemos a sua atenção e consideração.

Atenciosamente,
Antonio Pedro
Angola Open-source Community

**Comment by henriquemoody** [comment]
Ola @antonio-pedro99!

Em primeiro lugar, me desculpe pela demora em te responder. Eu fiquei muito tempo desconectado do open-source, mas voltem recentemente.

O Angola Open-Source Community parece um projeto bem interessante! Eu teria interesse em participar, entretanto, devo admitir que minha disponibilidade possa ser um pouco limitada nos próximos meses.

O que você acha de termos uma conversa por vídeo pra podemos conversar melhor sobre como eu posso contribuir? Te mandei uma connection request no LinkedIn. Vamos conversar!

**Comment by henriquemoody** [comment]
Vou fechar essa discussão, já que faz bastante tempo que ela está sem atividades.

Sinta-se a vontade pra abrir-la quando quiser 🐼

**Comment by antonio-pedro99** [comment]
Desculpe pela resposta tardia.

Sinta-se à vontade.
